src/pages/Landing/Landing.css

```diff
@@ -38,3 +37,74 @@
 .theme_Btn .bx {
 	font-size: 4rem;
 }
+
+/* Media Query for Landing Page */
+
+@media only screen and (max-width: 900px) {
+}
+
+@media only screen and (max-width: 780px) {
+
+	body .hero__mobile {
+		display: block;
+	}
+
+	body .hero__desktop {
+		display: none;
+	}
+
+	body .auth_btn {
+		padding: 8px 4rem;
+		font-size: 2rem;
+		margin-left: 0rem;
+	}
+
```

Comment:
not a good way for using html elem for specificity!

Reply:
Yeah that is to taken care of once i am done with all the functionality. Thanks for your review